doc/classes/AudioStreamPlayer.xml

```diff
@@ -111,26 +115,33 @@
 			<return type="void">
 			</return>
 			<description>
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game.
 		</member>
 		<member name="bus" type="String" setter="set_bus" getter="get_bus">
+			Bus on which this audio is playing.
 		</member>
 		<member name="mix_target" type="int" setter="set_mix_target" getter="get_mix_target" enum="AudioStreamPlayer.MixTarget">
 		</member>
 		<member name="playing" type="bool" setter="_set_playing" getter="is_playing">
+			If [code]'true'[/code], audio is playing.
 		</member>
 		<member name="stream" type="AudioStream" setter="set_stream" getter="get_stream">
+			The [AudioStream] object to be played.
 		</member>
 		<member name="volume_db" type="float" setter="set_volume_db" getter="get_volume_db">
+			Volume of sound, in dB.
 		</member>
 	</members>
 	<signals>
 		<signal name="finished">
 			<description>
+				Fires when the audio stops playing.
```

Comment:
Use "emitted", not "fires".

Reply:
I also see both are used in other files. Should we use only 'emitted'? I'll add that to my checklist.

Reply:
I've been removing them when I find them. Sticking with engine terminology - signals use `emit` not `fire`.